ds-streamlit/files/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM registry.access.redhat.com/ubi8/python-39
```

Comment:
I suggest using ubi9 image as per latest changes also in https://github.com/opendevstack/ods-quickstarters/blob/master/be-python-flask/files/docker/Dockerfile#L1